code/datums/craft/gun_parts.dm

```diff
@@ -258,27 +261,28 @@
 	icon_state = "barrel_35"
 	generic = FALSE
 	bad_type = /obj/item/part/gun/barrel
-	matter = list(MATERIAL_PLASTEEL = 4)
+	matter = list(MATERIAL_PLASTEEL = 8,)
```

Comment:
Does this comma work here?

Reply:
or in other matter lists?

Reply:
Forgot to test the recycling of parts. Yeah, fixing commas. All the crafting works, tested each single part

Reply:
glad to hear.
Looks good, otherwise